Block/Service/MenuBlockService.php

```diff
@@ -83,7 +83,7 @@ public function execute(BlockContextInterface $blockContext, Response $response
             'context' => $blockContext,
         ];
 
-        if ('private' === $blockContext->getSettings('cache_policy')) {
+        if ('private' === $blockContext->getSetting('cache_policy')) {
```

Comment:
lol nice

Reply:
This fixes a bug, right? It's not just style...

Reply:
Yes this is indeed a bugfix!